components/Footer.tsx

```diff
@@ -53,9 +52,11 @@ const Discalimer = () => (
 )
 
 const SubmitButton = () => (
-  <button className="bg-white text-black py-1 px-14 col-span-2 col-end-7 mb-4 flex items-center justify-center">
-    Submit an entry <ArrowRight className="ml-2" size={16} />
-  </button>
+  <a href="https://airtable.com/shru3ZGjdyhEGTzx6" target="_blank" rel="noreferrer" className="bg-white text-black py-1 px-14 col-span-2 col-end-7 mb-4 flex items-center justify-center">
+    <button>
+      Submit an entry <ArrowRight className="ml-2" size={16} />
+    </button>
+  </a>
```

Comment:
I think this should just be an anchor if it's an external link, not a button wrapped in an anchor. That's my bad for setting as a button in the first place sorry!